docs/shared/recipes/repo-types/integrated-in-package-based.md

```diff
@@ -0,0 +1,34 @@
+# Add an Integrated Project in a Package-based Repository
+
+In a package-based repository, you've intentionally opted out of some of Nx's features in order to give yourself more flexibility. If you decide to add a new project using the integrated style, the process is straight-forward.
+
+To add an integrated project:
+
+1. Install the plugin you want to use (i.e. `npm install @nx/react`)
+2. Generate an application using that plugin (i.e. `nx g @nx/react:app`)
+
+The integrated project is now ready to use. Next, we'll discuss some of the changes that were applied to your codebase.
+
+## package.json dependencies
+
+All the dependencies that are required for an integrated project are maintained in the root `package.json` file. In order to use the plugin's [Automate Updating Framework Dependencies](/core-features/automate-updating-dependencies) feature, we recommend using a [Single Version Policy](/more-concepts/dependency-management#single-version-policy) for integrated projects.
+
+## tsconfig.base.json
+
+Even if the project you added only uses javascript, a `tsconfig.base.json` file is generated in the root of the repo. This file is needed for the tooling that Nx provides.
```

Comment:
I'd expand this section explaining why this is the case. Basically due to how Nx plugins use TS path mappings rather than NPM/yarn/pnpm workspaces to manage local dependencies.